### PR TITLE
feat: badge component added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
feature of the badge component is added in this component library.
1.  Badge on icons is added.
2. badge on avatars is added.
3. and badges with three-digit numbers are also added.
Please review the code and give your sugggestion

### CODE REVIEW

File: component_documentation/componentDoc.html [hunk 1]
```diff
@@ -315,6 +315,95 @@ <h2 class="alert--heading">Application submitted!</h2>
...
+            </div>
+            <div class="badge">
+                <img src="/image/badge_image/whatsapp.png" alt="" class="badge__img" >
+                <span class="badge__text badge--two-unit" >11</span>
+            </div>
+        
+            <div class="badge">
+                <img src="/image/badge_image/whatsapp.png" alt="" class="badge__img" >
+                <span class="badge__text badge--three-digit" >101</span>
+            </div>
```
Comment (rkapoor10) [inline component_documentation/componentDoc.html:338]: don't leave the alt property empty

File: component_documentation/componentDoc.html [hunk 2]
```diff
@@ -315,6 +315,95 @@ <h2 class="alert--heading">Application submitted!</h2>
...
+           </div>
+           <div class="component-code">
+               <div class="code-snippet-container ">
+                   <textarea class="code-snippet" name="code-snippet" rows="5" cols="" readonly>
+<div class="badge">
+        <img src="/image/badge_image/email.png" alt="" class="badge__img" >
```
Comment (rkapoor10) [inline component_documentation/componentDoc.html:344]: same as above

File: componentLibrary.css [hunk 3]
```diff
@@ -119,4 +119,29 @@
...
+
+.badge {
+    position: relative;
+    display: inline-block;
+    margin: 1rem;
+}
+
+.badge__text {
+    position: absolute;
+    background-color: red;
```
Comment (rkapoor10) [inline componentLibrary.css:134]: you may use variables for colors

**Review by rkapoor10 — APPROVED** [review]
Liked the three-digit badge feature. Make the changes and it's good to merge.

**Review by mariyasada — APPROVED** [review]
you can merge it

File: component_documentation/componentDoc.html (hunk 1 above)
Comment (ShreyashMahajan) [reply component_documentation/componentDoc.html:338]: ok, i will change it, Thank you

File: component_documentation/componentDoc.html (hunk 2 above)
Comment (ShreyashMahajan) [reply component_documentation/componentDoc.html:344]: ok

File: componentLibrary.css (hunk 3 above)
Comment (ShreyashMahajan) [reply componentLibrary.css:134]: ok

**Comment by ShreyashMahajan** [comment]
> you can merge it

thank you

**Pull request merged**